src/material-experimental/mdc-progress-bar/progress-bar.ts

```diff
@@ -227,6 +227,10 @@ export class MatProgressBar
       )
         .pipe(filter((e: TransitionEvent) => e.target === this._primaryBar))
         .subscribe(() => {
```

Comment:
There's a similar listener in the non-MDC progress bar where we should have a similar check.